README.md

```diff
@@ -1,5 +1,5 @@
 # Synergy Design System
-<img src="./packages/docs/public/synergy-logo-light.png" alt="Synergy-Logo" width="400"/><br>
+<img src="./packages/docs/public/synergy_logo_transparent.png" alt="Synergy-Logo" width="400"/><br>
```

Comment:
Please provide logo in light and dark mode:

https://www.stefanjudis.com/notes/how-to-define-dark-light-mode-images-in-github-markdown/

Otherwise it looks like this:
![CleanShot 2023-12-11 at 09 23 15@2x](https://github.com/synergy-design-system/synergy-design-system/assets/26542182/67d4c735-a680-4fb5-91f4-b09d29be310e)